README.md

```diff
@@ -105,40 +105,42 @@ reek --help
+1. Using the CLI `-c` switch (see [_Command-line interface_](#command-line-interface) above)
+2. Having a file ending with `.reek` either in your current working directory or in a parent directory (more on that later)
+3. Having a file ending with `.reek` in your `$HOME` directory
 
-The order in which `reek` tries to find such a configuration file is exactly like above: First `reek` checks if we have given it a configuration file explicitly via CLI. Then it checks the current working directory for a file and if it can't find one, it traverses up the directories until it hits the root directory. And lastly, it checks your HOME directory.
+The order in which reek tries to find such a configuration file is exactly the above: first it checks if we have given it a configuration file explicitly via CLI; then it checks the current working directory for a file and if it can’t find one, it traverses up the directories until it hits the root directory; lastly, it checks your `$HOME` directory.
 
-As soon as `reek` detects a configuration file it stops searching immediately, meaning that from `reek`'s point of view there exists one configuration file and one configuration only regardless of how many ".reek" files you might have on your filesystem.
+As soon as reek detects a configuration file it stops searching immediately, meaning that from reek’s point of view there exists exactly one configuration file and one configuration, regardless of how many `*.reek` files you might have on your filesystem.
 
 #### Configuration options
 
 The first thing you probably want to check out are the [Basic Smell Options](https://github.com/troessner/reek/wiki/Basic-Smell-Options) which are supported by every smell type.
-Certain smell types offer a configuration that goes beyond that of the basic smell options - for instance [Data Clump](https://github.com/troessner/reek/wiki/Data-Clump).
-All options that go beyond the [Basic Smell Options](https://github.com/troessner/reek/wiki/Basic-Smell-Options) should be documented in the corresponding smell type wiki page but if you want to get a quick and full overview over all possible configurations you can always check out [the default.reek file in this repository](https://github.com/troessner/reek/blob/master/config/defaults.reek).
+Certain smell types offer a configuration that goes beyond that of the basic smell options – for instance [Data Clump](https://github.com/troessner/reek/wiki/Data-Clump).
+All options that go beyond the [Basic Smell Options](https://github.com/troessner/reek/wiki/Basic-Smell-Options) should be documented in the corresponding smell type wiki page, but if you want to get a quick and full overview over all possible configurations you can always check out [the `config/default.reek` file in this repository](config/defaults.reek).
 
-Here's an excerpt of a `reek` configuration file from a commercial project:
+Here’s an excerpt of a reek configuration file from a commercial project:
```

Comment:
Digging a little deeper, I think all these single right quote characters should be reverted to apostrophes.